diff --git a/src/devices/cpu/cpudevice.cpp b/src/devices/cpu/cpudevice.cpp
index c792a76d..bc809136 100644
--- a/src/devices/cpu/cpudevice.cpp
+++ b/src/devices/cpu/cpudevice.cpp
@@ -2646,7 +2646,8 @@ namespace fastllm {
         
         // 为每个线程分配任务状态
         for (int i = 0; i < numThreads; i++) {
-            taskStates[i] = new (std::align_val_t{64}) TaskState();
+            void *taskStateMem = ::operator new(sizeof(TaskState), std::align_val_t{64});
+            taskStates[i] = new (taskStateMem) TaskState();
             taskStates[i]->curr.store(0, std::memory_order_relaxed);
             taskStates[i]->end = 0;
             taskStates[i]->completed.store(false, std::memory_order_relaxed);
diff --git a/src/models/chatglm.cpp b/src/models/chatglm.cpp
index 8e05aaa1..3588fec6 100644
--- a/src/models/chatglm.cpp
+++ b/src/models/chatglm.cpp
@@ -968,7 +968,7 @@ namespace fastllm {
                 return input;
             } else {
 #if defined(_WIN32) or defined(_WIN64)
-                return history + ("[Round " + std::to_string(round) + u8"]\n\n问：" + input + u8"\n\n答：");
+                return history + ("[Round " + std::to_string(round) + "]\n\n问：" + input + "\n\n答：");
 #else
                 return history + ("[Round " + std::to_string(round) + "]\n\n问：" + input + "\n\n答：");
 #endif
@@ -984,7 +984,7 @@ namespace fastllm {
 		if (GetVersion() == 2)
 			round++;
 #if defined(_WIN32) or defined(_WIN64)
-        return (history + ("[Round " + std::to_string(round) + u8"]\n\n问：" + input + u8"\n\n答：" + output + "\n"));
+        return (history + ("[Round " + std::to_string(round) + "]\n\n问：" + input + "\n\n答：" + output + "\n"));
 #else
         return (history + ("[Round " + std::to_string(round) + "]\n\n问：" + input + "\n\n答：" + output + "\n\n"));
 #endif
